Sources/SwiftDocC/Model/Rendering/RenderNodeTranslator.swift

```diff
@@ -622,54 +622,75 @@ public struct RenderNodeTranslator: SemanticVisitor {
             node.primaryContentSections.append(ContentRenderSection(kind: .content, content: discussionContent, heading: title))
         }
         
-        if let topics = article.topics, !topics.taskGroups.isEmpty {
-            // Don't set an eyebrow as collections and groups don't have one; append the authored Topics section
-            node.topicSections.append(
-                contentsOf: renderGroups(
-                    topics,
-                    allowExternalLinks: false,
-                    allowedTraits: documentationNode.availableVariantTraits,
-                    contentCompiler: &contentCompiler
+        node.topicSectionsVariants = VariantCollection<[TaskGroupRenderSection]>(
+            from: documentationNode.availableVariantTraits,
+            fallbackDefaultValue: []
+        ) { trait in
+            var sections = [TaskGroupRenderSection]()
+            
+            if let topics = article.topics, !topics.taskGroups.isEmpty {
+                // Don't set an eyebrow as collections and groups don't have one; append the authored Topics section
```

Comment:
Hmm, that's an interesting edge case. I'm not convinced that API collection pages should only be available in the language in which their topics are curated, because the API collection page could contain prose content that's interesting regardless of the language. It's easy to create an API collection accidentally (i.e., by just adding topics to an Article). Maybe we can move this conversation to the forums. Separate from that, this would be technically challenging because we'd need to parse the contents of the page before assigning its available languages.

Reply:
Yeah that's fair. Maybe the right default behavior is that we should turn it into an Article in one language and an API collection page in the other.

Then as a future enhancement we should add a metadata directive that limits a given page to one language or the other.

Reply:
> Then as a future enhancement we should add a metadata directive that limits a given page to one language or the other.

I like that idea. A directive that forces the languages in which an article (and framework, maybe?) is available. Something to discuss in more detail in the forums.

Reply:
I filed https://bugs.swift.org/browse/SR-15998 to track this.